packages/react-scripts/scripts/build.js

```diff
@@ -79,12 +79,12 @@ checkBrowsers(paths.appPath, isInteractive)
   .then(() => {
     // First, read the current file sizes in build directory.
     // This lets us display how much they changed later.
-    return measureFileSizesBeforeBuild(paths.appBuild);
+    return measureFileSizesBeforeBuild(paths.appBuildWeb);
```

Comment:
What happens if SSR is not on? The build won't be in this path if this code is what we go with - might need to check that for all paths changed in this file to thise new path

https://github.com/Skyscanner/backpack-react-scripts/pull/109/files#diff-8e25c4f6f592c1fcfc38f0d43d62cbd68399f44f494c1b60f0cf9ccd7344d697R268

Reply:
Have made it flexible for both CSR and SSR.